app/controllers/application_controller.rb

```diff
@@ -87,7 +87,9 @@ def authenticate_user_from_token!
 
   # This provides IP whitelisting on top of token authentication for added security.
   def authenticate_internal_user_from_token!
```

Comment:
I think this should be a PR to the branch "gdingle/action-cache"? I.e. " gregdingle wants to merge 1 commit into gdingle/action-cache from gdingle/fix-sample-viewable"? `authenticate_internal_user_from_token!` no longer exists in master because I reverted your last commit

Reply:
Yea, I wasn't planning on merging this at all, only deploy it to staging, but if I did merge, it woudl be on `gdingle/action-cache`.